src/web/app/pages-admin/admin-search-page/admin-search-page.component.ts

```diff
@@ -64,6 +65,10 @@ export class AdminSearchPageComponent {
         this.students = resp.students;
         this.hideAllInstructorsLinks();
         this.hideAllStudentsLinks();
+        if (this.instructors.length + this.students.length >= this.maxResultsToShow) {
```

Comment:
Seems like there is quite a bit of nesting of if-statements, any way to clean it up a bit?

Reply:
I shifted the if-statement up a level, is that better?

Reply:
Yup it is better.